src/components/TVLAreaChart.js

```diff
@@ -100,7 +103,7 @@ function TVLAreaChart({
         : fuseFormatter.format(lastDataValue),
       date: lastData ? lastData.date : 0,
     });
-  }, [useUSD]);
+  }, [filteredData, useUSD]);
```

Comment:
this was the problematic line that hid the numbers, since overlay is not updated based on data updates... they were not being summed after calculated